src/app/register/register.component.ts

```diff
@@ -152,7 +163,8 @@ export class RegisterComponent implements OnInit, OnDestroy, AfterViewInit {
             next: () => {
               // We can't re-use a captcha code twice, so auto-login won't work right now. For now we just
               // redirect to the login form
-              window.location.href = "/login";
+              const newPath = "/login" + '?r=' + encodeURIComponent(this.redirectPath);
+              window.location.href = newPath;
```

Comment:
I think it's likely to be more consistent with our other navigation and perhaps lighter on the browser to use the `router` that's already injected so that Angular can potentially update just what's needed in the already loaded app. Not a blocker.

Reply:
Thanks, I think you're probably right but will keep it as-is since that's closer to the existing working code, and I don't want to spend more time than necessary on this before CC.